packages/cli/src/adapters/locked-gold.ts

```diff
@@ -61,13 +61,13 @@ export class BondedDepositAdapter {
     }
   }
 
-  async getBondedDepositValue(account: string, noticePeriod: string) {
+  async getLockedCommitmentValue(account: string, noticePeriod: string) {
```

Comment:
How about `getLockedCommitmentValue` -> `getLockedGoldCommitmentValue` for clarity?

Reply:
I like it

Reply:
actually, I think the context is already clear. this is in a `LockedGold` package (as it is in `LockedGold.sol` too) and here the use of `Locked` is like `Bonded` -- contrast `getNotifiedCommitmentValue`